### PR TITLE
Chrome-HTTP

### PR DESCRIPTION
**Pull request by bnandaku**
Refer #78

### CODE REVIEW

File: public/javascripts/click2vox-1.5.0.js [hunk 1]
```diff
@@ -598,7 +598,13 @@ var check1Ready = (function() {
   // Click on Make Call button event
   handleEvent('click', '.vxb-widget-box #launch_call', function (e) {
     e.preventDefault();
-    makeCall();
+    if(!isChromeOnHttp()){
+      makeCall();
+    } else {
+      var buttonData = document.querySelector('.voxButton');
+      // console.log(buttonData.dataset);
```
Comment (san983) [inline public/javascripts/click2vox-1.5.0.js]: Please remove this line

File: routes/widget.js [hunk 2]
```diff
@@ -88,6 +88,19 @@ router.post('/portal-widget/get-code', function(req, res, next) {
   }
 });
 
+router.post('/portal-widget/get-html', function(req, res, next){
```
Comment (san983) [inline routes/widget.js]: Please add a comment here for what this endpoint is used

File: routes/utils.js [hunk 3]
```diff
@@ -50,5 +50,20 @@ module.exports = {
     };
 
     return jade.renderFile('./views/voxbone_widget_div.jade', params);
+  },
+
+  widgetSecureDivhtml: function (widget, did) {
```
Comment (san983) [inline routes/utils.js]: Please rename to `widgetSecureDivHTML`

File: public/javascripts/click2vox-1.5.0.js [hunk 4]
```diff
@@ -704,5 +710,22 @@ var check1Ready = (function() {
   init();
 });
 
+open = function(verb, url, data) {
```
Comment (san983) [inline public/javascripts/click2vox-1.5.0.js]: Please rename `open` to something more specific, like `openPopup` or so

**Comment by bnandaku** [comment]
@san983 Made the requested changes

**Pull request merged**